integration_test/workflow_tests/run_tests.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import testtools
+
+import test_packages
+
+
+class TracingStreamResult(testtools.StreamResult):
+    def __init__(self):
+        self.test_results = {}
+
+    def status(self, *args, **kwargs):
+        self.test_results[kwargs["test_id"]] = kwargs["test_status"]
+
+
+def run_tests():
+    suite = unittest.TestLoader().loadTestsFromModule(test_packages)
+    concurrent_suite = \
+        testtools.ConcurrentStreamTestSuite(lambda: ((case, None) for case in suite))
+    result = TracingStreamResult()  # testtools.StreamResult()
+    result.startTestRun()
+    concurrent_suite.run(result)
+    result.stopTestRun()
+
+    print(f"Checking results on {len(result.test_results)} tests...")
+    failed_tests = []
+    for test_name in result.test_results:
+        if result.test_results[test_name] != 'success':
+            failed_tests.append(test_name)
+
+    if any(failed_tests):
```

Comment:
Other than listing the failed tests, do we have any logging or information that tells us why they failed? Maybe from the test themselves?

Reply:
Anything on standard output, including assertion failures.

Reply:
Standard output, though with the recommended use of logger much is hidden unless the level is set lower.

Reply:
Added some try-catch logging, as apparently standard unit test logic does not write failures to stdout.